### PR TITLE
HB-5933: use os_log instead of print

### PR DESCRIPTION
**Pull request by bwised**
https://chartboost.atlassian.net/browse/HB-5933

### CODE REVIEW

**Review by daniel-barros — DISMISSED** [review]
LGTM

File: Source/AdColonyAdapterConfiguration.swift [hunk 1]
```diff
@@ -4,18 +4,23 @@
 // license that can be found in the LICENSE file.
 
 import AdColony
+import os.log
 
 /// A list of externally configurable properties pertaining to the partner SDK that can be retrieved and set by publishers.
 @objc public class AdColonyAdapterConfiguration: NSObject {
 
+    private static var log = OSLog(subsystem: "com.chartboost.mediation.adapter.adcolony", category: "Configuration")
```
Comment (daniel-barros) [inline Source/AdColonyAdapterConfiguration.swift:12]: nit: could be a let
```suggestion
    private static let log = OSLog(subsystem: "com.chartboost.mediation.adapter.adcolony", category: "Configuration")
```

File: Source/AdColonyAdapterConfiguration.swift [hunk 2]
```diff
@@ -4,18 +4,23 @@
...
     /// Flag that can optionally be set to enable the partner's test mode.
     /// Disabled by default.
     @objc public static var testMode: Bool = false {
         didSet {
             let options = AdColonyAdapter.options
             options.testMode = testMode
             AdColony.setAppOptions(options)
-            print("AdColony SDK test mode set to \(testMode)")
+            if #available(iOS 12.0, *) {
+                os_log(.debug, log: log, "AdColony SDK test mode set to %{public}s", testMode ? "true" : "false")
```
Comment (daniel-barros) [inline Source/AdColonyAdapterConfiguration.swift:22]: nit: not sure how this works, but would it be straightforward to use the Bool default description like we were doing in the print?
E.g. something like `os_log(.debug, log: log, "AdColony SDK test mode set to %{public}s", testMode)`
maybe changing the `%{}s` to something else idk

Comment (bwised) [inline Source/AdColonyAdapterConfiguration.swift:22]:
```suggestion
                os_log(.debug, log: log, "AdColony SDK test mode set to %{public}s", "\(testMode)")
```

File: Source/AdColonyAdapterConfiguration.swift [hunk 3]
```diff
@@ -26,7 +31,9 @@ import AdColony
             let options = AdColonyAdapter.options
             options.disableLogging = !verboseLogging
             AdColony.setAppOptions(options)
-            print("AdColony SDK verbose logging set to \(verboseLogging)")
+            if #available(iOS 12.0, *) {
+                os_log(.debug, log: log, "AdColony SDK verbose logging set to %{public}s", verboseLogging ? "true" : "false")
```
Comment (bwised) [inline Source/AdColonyAdapterConfiguration.swift:35]:
```suggestion
                os_log(.debug, log: log, "AdColony SDK verbose logging set to %{public}s", "\(verboseLogging)")
```

**Comment by bwised** [comment]
I'll make the same changes on the other PRs.

**Review by daniel-barros — APPROVED** [review]

**Pull request merged**